test/parallel/test-buffer-alloc.js

```diff
@@ -1060,6 +1060,17 @@ assert.throws(function() {
   Buffer.allocUnsafe(0xFFFFFFFFF);
 }, RangeError);
 
+// issue GH-9226
+{
+  const buf = Buffer.alloc(4, 'YQ==', 'base64');
+  const expectedBuf = new Buffer([97, 97, 97, 97]);
```

Comment:
Good idea; I had been under the impression that `Buffer.from()` had different behavior in v4, but it seems like that's not an issue here.